src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -79,12 +73,7 @@ public async Task InvokeAsync(HttpContext context)
             {
                 if (!MediaTypeHeaderValue.TryParse(httpRequest.ContentType, out var mediaTypeHeader))
                 {
-                    await WriteErrorResponseAsync(
-                        httpResponse,
-                        serializer,
-                        cancellationToken,
-                        $"Invalid 'Content-Type' header: value '{httpRequest.ContentType}' could not be parsed.",
-                        HttpStatusCode.BadRequest);
```

Comment:
Changed to media not supported error code.

Reply:
👍